generators/service-autoscaling/templates/swift/instrumentation.swift

```diff
@@ -1,13 +1,14 @@
 import LoggerAPI
+import CloudEnvironment
 import SwiftMetricsBluemix
 
-var autoscaling: SwiftMetricsBluemix?
-
-func initializeServiceAutoscaling() throws {
+func initializeServiceAutoscaling(cloudEnv: CloudEnv) throws -> SwiftMetricsBluemix? {
+    var autoscaling: SwiftMetricsBluemix? = nil
     guard let swiftMetrics = swiftMetrics else {
         Log.warning("Failed to initialize Auto-Scaling because metrics is not initialized.")
-        return
+        return autoscaling
```

Comment:
Would be clearer to `return nil` here and move the declaration of the `autoscaling` variable down below the guard (making it a non-optional `let` in the process).

Reply:
done